telos-frontend/src/views/journal/Journal.jsx

```diff
@@ -28,21 +28,25 @@ const Journal = () => {
   // widget will be added on the right page if isRight is true
   const [isRight, setIsRight] = useState(true);
 
+  // Iterate the dates at the top of the journal pages by -1: to be used when the left 'back' arrow button being pressed
```

Comment:
Instead of "-1", would "the previous day" or similar make more sense?
Also, is it necessary to include how the method would be used, e.g. it could be used in a different context if we had other buttons?

Reply:
Yes both of these points make a lot of sense, I will update based on this, thank you! :)